WordPress/Classes/Utility/App Configuration/AppConstants.swift

```diff
@@ -48,6 +48,10 @@ extension AppConstants {
         static let workWithUsURL = "https://make.wordpress.org/mobile/handbook"
     }
 
+    struct AppRatings {
+        static let prompt = NSLocalizedString("What do you think about WordPress?", comment: "This is the string we display when prompting the user to review the app")
```

Comment:
[Nitpick]: [Include a key](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys) with the new strings. Probably isn't that big of deal in this context since I doubt anywhere else will use these strings.

Reply:
Good callout @wargcm! 🙇 

I added keys for all updated strings [in this commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/20276/files/29b1f4297910e2fe81f7ad155975df628de9e8e5..0ee205b2c1173f85b4220f15b27a2c1c230e20f9). Please feel free to critique the keys chosen!

Reply:
Looks good to me!